types/tinymce/index.d.ts

```diff
@@ -633,7 +633,7 @@ export interface Formatter {
 
   matchNode(node: html.Node, name: string, vars: {}, similar: boolean): {};
 
-  register(name: {}, format?: {}): void;
+  register(name: {} | string, format?: {} | object[]): void;
```

Comment:
Got it. Did not know about that equivalency. Since the current implementation satisfies what my MR was trying to accomplish, I guess we are good here. I will drop this MR.

Thanks for the explanation